.github/workflows/cosi-helm.yaml

```diff
@@ -0,0 +1,29 @@
+name: COSI driver
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: ["**"]
+
+env:
+  GOPRIVATE: github.com/dell/*
+  TOKEN: ${{ secrets.GH_DELL_ACCESS }}
+
+jobs:
+  kube-linter:
```

Comment:
There is an existing linter used for all charts located here: https://github.com/dell/helm-charts/blob/main/.github/workflows/helm-validations.yml#L29C6-L29C6

This file may no longer be required or that existing helm-validations.yml file could be updated if this new kube-linter-action is more up to date than what we are using in our existing workflow.

Reply:
As you can see here: https://github.com/dell/helm-charts/actions/runs/5999804015/job/16270649466?pr=267, after adding scanning to the whole repo many issues in other charts appeared. 

I think that in this PR the cosi chart should be scanned, but other teams should correct their charts. The additional tasks have to be planned.

Reply:
@tdawe To add my 5 cents, I think there should be 2 parallel configurations for KubeLinter, as the rules applicable to things like COSI Driver or CS Modules, might not be applicable to CSI Drivers, as CSI Drivers are interacting with K8s Node's filesystem.